inginious/frontend/templates/course_user_settings.html

```diff
@@ -0,0 +1,77 @@
+                <i class="fa fa-group fa-fw"></i>&nbsp;
+                {% set mygroup = user_manager.get_course_user_group(course) %}
+                {% if mygroup and user_manager.session_username() in mygroup['students'] %}
+                    {{ _("Group : {}").format(mygroup['description']) }}
+                {% else %}
+                    {{ _("Group management") }}
+                {% endif %}
+            </a>
+            <a class="list-group-item list-group-item-action list-group-item-info" href="{{ get_homepath() }}/user_settings/{{ course.get_id() }}">
+                <i class="fa fa-cogs fa-fw"></i>&nbsp; {{ _("Course Settings") }}
+            </a>
+        </div>
+    {%endif %}
+{% endblock %}
+{% block navbar %}
+    <nav aria-label="breadcrumb">
+        <ol class="breadcrumb">
+            {% if registered %}
+                <li class="breadcrumb-item"><a href="{{ get_homepath() }}/mycourses" title="{{ _('My courses') }}" data-toggle="tooltip" data-placement="bottom"><i class="fa fa-th-list"></i></a></li>
+            {% else %}
+                <li class="breadcrumb-item"><a href="{{get_homepath() }}/courselist" title="{{ _('Course list') }}" data-toggle="tooltip" data-placement="bottom"><i class="fa fa-th-list"></i></a></li>
+            {% endif %}
+            <li class="breadcrumb-item"><a href="{{get_homepath() }}/course/{{ course.get_id() }}">{{ course.get_name(user_manager.session_language()) }} <span class="sr-only">{{ _("(current)") }}</span></a></li>
+            <li class="breadcrumb-item active"><a href="#"><i class="fa fa-cogs"></i> {{ _("Course settings") }}</a></li>
+        </ol>
+    </nav>
+{% endblock %}
+{% block content %}
+    {%  if feedback is not none %}
+        <div class="alert alert-{{ feedback[0] }}" role="alert">
+            {{ _(feedback[1] ) }}
+        </div>
+    {% endif %}
+    <h2>{{_("Course settings")}}</h2>
+    <form method="post" action="">
+    {% for key in course_user_setting_fields %}
+        <div class="form-group row">
+            <label class="col-sm-2 " for="{{ key }}" class="control-label">{{ key }}</label>
+            {% if  course_user_setting_fields[key].get_type_name() == 'BOOLEAN' %}
+                <label><input class="" type="checkbox" id="{{ key }}" name="{{ key }}" {% if course_user_settings[key]%} checked {% endif %}>{{ course_user_setting_fields[key].get_description() }}</label>
+            {% else %}
+                <input id="{{ key }}" name="{{ key }}" placeholder="{{ course_user_setting_fields[key].get_description() }}"
+                        {% if course_user_setting_fields[key].get_type_name() == 'INTEGER' %}
+                            type="number"
+                        {% else %}
+                            type="text"
+                        {% endif %}
+                       class="form-control col-sm-10 " value="{{ course_user_settings[key] }}">
```

Comment:
You should not apply `col-sm-10` with `form-control` but wrap the `<input/>` with a `</div/>`, otherwise the input field goes beyond the margins of its container